transcom/mymove
# B 20244 int

## [B-20244](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A966366)

## Summary

Fixing issue where prime V3 documentation did not display.

https://github.com/transcom/mymove-docs

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

https://github.com/transcom/mymove-docs

1. Run the project
2. Pull down the mymove-docs project
3. Run the mymove-docs project, and follow instructions in above readme to display your local files
4. Verify prime v3 displays

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
